plugins/humanitec/dynamic-export/export-dynamic-plugins.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/sh
+
+currentDir=$(cd $(dirname "$0") && pwd)
+echo ===================================
+echo Exporting frontend plugin
+echo ===================================
+cd $currentDir/..
+npx --yes @janus-idp/cli@1.7.10 package export-dynamic-plugin --embed-as-dependencies --clean
+mkdir -p $currentDir/output
+npm pack ./ --pack-destination $currentDir/output
```

Comment:
@davidfestal do we need to publish this version of the frontend plugin to make it work with RHDH?